octobot/cli.py

```diff
@@ -232,6 +234,10 @@ def octobot_parser(parser):
                         help='On multiple files backtesting: run on the whole available data instead of the '
                              'common part only (default behavior).',
                         action='store_true')
+    parser.add_argument('-ebt', '--enable-backtesting-timeout',
+                        help='To disable when running long backtesting : disable backtesting watcher timeout '
+                             'that prevent backtesting from being interrupted during execution.',
```

Comment:
Limit the backtesting run to backtesting_constants.BACKTESTING_DEFAULT_JOIN_TIMEOUT /60 minutes.